style.css

```diff
@@ -1560,4 +1560,12 @@ body.vacancy-page #empty_vacancy{
 	height: 9rem;
 
 }
+
+.warning {
+	color: maroon;
+    font-weight: 400;
+    font-size: 0.9em;
+    padding-bottom: 1em;
+    padding-top: 1em;
```

Comment:
```suggestion
	color: maroon;
	font-weight: 400;
	font-size: 0.9em;
	padding-bottom: 1em;
	padding-top: 1em;
```

Reply:
I did not bother with proper spacing since spacing is all over the place already :p